website/docs/r/policy_l2_vpn_service.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+subcategory: "Policy - VPN"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_l2_vpn_service"
+description: A resource to configure a L2 VPN Service.
+---
+
+# nsxt_policy_l2_vpn_service
+
+This resource provides a method for the management of a L2 VPN Service.
+
+This resource is applicable to NSX Policy Manager and VMC.
+
+## Example Usage
+
+```hcl
+resource "nsxt_policy_l2_vpn_service" "test" {
+  display_name          = "l2-vpn-service1"
+  description           = "Terraform provisioned L2 VPN service"
+  locale_service_path   = /infra/tier-0s/aaa/locale-services/default
```

Comment:
We normally don't suggest to use absolute path strings in documentation, but rather things like `data.nsxt_policy_gateway_locale_service.path`. Regardless, string needs to be quoted.

Reply:
done